packages/dataviews/src/style.scss

```diff
@@ -243,10 +250,10 @@
 		white-space: nowrap;
 		overflow: hidden;
 		display: block;
-		width: 100%;
+		flex-grow: 0;
```

Comment:
Not sure if this happens as a result of this change, but here's what I see for the focus ring:
<img width="720" alt="Captura de ecrã 2024-03-14, às 12 06 26" src="https://github.com/WordPress/gutenberg/assets/583546/518f0107-431f-4575-b6bb-0c27ff92226c">

Reply:
https://github.com/WordPress/gutenberg/pull/60191